### ISSUE TITLE
Add core block stylesheets to Reader mode template

### ISSUE THREAD

**Issue by westonruter** [ampproject/amp-wp#4204]
## Feature description

When blocks are added to post content they can appear very poorly in Reader mode. The reason for this is the block styles from core are not being included in the template. Before #2202 this was not feasible to do due to the lack of tree shaking, but now that it is resolved this is no longer the case. 

The Reader mode templates should be made fully block-compatible by including the `wp-block-library` stylesheet in the template. What's more is that the templates should behave as if `current_theme_supports( 'wp-block-styles' )` so that `wp-block-library-theme` is also added. In fact, the Reader mode templates should probably go ahead and call `wp_common_block_scripts_and_styles()` so that the `enqueue_block_assets` action is triggered. Any stylesheet that is enqueued as part of that should then get printed.

Related: #2044

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

The basic implementation can look like this:

```diff
--- a/templates/html-start.php
+++ b/templates/html-start.php
@@ -40,6 +40,7 @@
 	<style class="style-template-part">
 		<?php $this->load_parts( [ 'style' ] ); ?>
 	</style>
+	<?php wp_print_styles( [ 'wp-block-library', 'wp-block-library-theme' ] ); ?>
 	<style class="amp-post-template-css-action">
 		<?php
 		/**
```

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

**Comment by westonruter** [comment]
Fixed via #4299.

**Comment by csossi** [comment]
Verified in QA
